src/libraries/Microsoft.Extensions.Configuration/src/ConfigurationManager.cs

```diff
@@ -14,12 +14,15 @@
 namespace Microsoft.Extensions.Configuration
 {
     /// <summary>
-    /// ConfigurationManager is a mutable configuration object. It is both an <see cref="IConfigurationBuilder"/> and an <see cref="IConfigurationRoot"/>.
-    /// As sources are added, it updates its current view of configuration.
+    /// Represents a mutable configuration object.
     /// </summary>
+    /// <remarks>
+    /// It is both an <see cref="IConfigurationBuilder"/> and an <see cref="IConfigurationRoot"/>.
+    /// As sources are added, it updates its current view of configuration.
+    /// </remarks>
     [DebuggerDisplay("{DebuggerToString(),nq}")]
     [DebuggerTypeProxy(typeof(ConfigurationManagerDebugView))]
-    public sealed class ConfigurationManager : IConfigurationBuilder, IConfigurationRoot, IDisposable
+    public sealed class ConfigurationManager : IConfigurationManager, IConfigurationBuilder, IConfigurationRoot, IDisposable
```

Comment:
Can't `IConfigurationBuilder` be removed now since it comes transitively from `IConfigurationManager` now?
```suggestion
    public sealed class ConfigurationManager : IConfigurationManager, IConfigurationRoot, IDisposable
```